src/components/CertificateTemplates/tlds/sg/edu/nus/NUS-K1-2019/sample.opencert

```diff
@@ -1,627 +1,627 @@
 {
   "schema": "opencerts/v1.5",
   "data": {
-    "id": "0dffcd2c-e2fa-430b-bac6-00664af9ede4:string:NUS-A0127136L-B021000-P10182357-1-20180630",
-    "$template": "1c2be247-03bb-4be7-bdc8-6d8e8b83d100:string:sg/edu/nus/NUS-K1-2019",
-    "issuedOn": "84b9c5dc-f432-4579-b1b9-6b9a8be1cf2f:string:2019-08-27T10:03:17+08:00",
-    "name": "cca84721-5985-4153-a710-7d7683f5a83f:string:NUS Degree",
+    "id": "5357daa4-7d70-4a85-a869-9c37dbe49f5d:string:NUS-A0127136L-B021000-P10182357-1-20180630",
+    "$template": "ab8b23bc-3dc2-4105-a912-973cb2d998a2:string:sg/edu/nus/NUS-K1-2019",
+    "issuedOn": "7c5de081-f0e6-4c6c-b0f5-23352f559869:string:2019-08-27T10:03:17+08:00",
+    "name": "c04e40e2-2ed3-4ebf-92b6-bcd842771428:string:NUS Degree",
     "issuers": [
       {
-        "name": "e4caaa03-eb14-45b1-9f9e-c71eeeac3150:string:National University of Singapore",
-        "did": "2ef3c376-66b5-40cd-b0c7-be2c586fa406:string:DID:SG-UEN:200604346E",
-        "url": "099db67c-3823-4e45-ad77-adbf91b4266f:string:https://www.nus.edu.sg",
-        "email": "ed0c5c4d-16c4-4fd1-9ffb-f7d4013905ac:string:registrar@nus.edu.sg",
-        "certificateStore": "9c83e80b-2cb8-4e74-a09a-7168b8bf84af:string:0xfEbB273495F5C2c4783E23424Fe9773691b57fcB"
+        "name": "97c24c4d-63df-44fb-9364-636a39c00d17:string:National University of Singapore",
+        "did": "324e631d-a2d9-43ab-98ca-b93bc7f1f5c8:string:DID:SG-UEN:200604346E",
```

Comment:
I missed this before, but could you use "uen" instead of "did" ? DID is a separate identification scheme which hasn't been rolled out yet

Reply:
ok change done.